runtime/rasdump/javadump.cpp

```diff
@@ -1244,6 +1244,47 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 		}
 	}
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
can you make a helper for the restore args and the normal vmArgs? It seems mostly the same code minus the header messages